### ISSUE TITLE
[CS 1.6] [Feature Request] Cvar? Serverside view model movements block

### ISSUE THREAD

**Issue by Droads2** [ValveSoftware/halflife#2626]
Is it possible to add the ability to disable certain players from the entire movement of the view model from the server side? Specifically, it is necessary for my mod, for example
In the first screenshot, the player is aiming straight.
https://ibb.co/fx7S6cc
In the second screenshot, the player aims to the bottom
https://ibb.co/6DmFkTG
There are many options for which such an opportunity could be used. 
But I don’t know if this is possible.

**Comment by SamVanheer** [comment]
If you're making a mod you can just modify the client code to not do that.

**Comment by Droads2** [comment]
> If you're making a mod you can just modify the client code to not do that.
This is serverside mod:) The main goal is that anyone can just go in and play without downloading any third-party client. But, unfortunately, some things cannot be implemented solely by the server side :(

**Comment by Droads2** [comment]
> > If you're making a mod you can just modify the client code to not do that.
This is serverside mod:) The main goal is that anyone can just go in and play without downloading any third-party client. But, unfortunately, some things cannot be implemented solely by the server side :(

**Comment by afwn90cj93201nixr2e1re** [comment]
So, you can create camera and use attach view.

**Comment by Droads2** [comment]
> So, you can create camera and use attach view.

I think, custom viewmodel on serverside is a bad idea, ping and prediction..
A good result even in fullpack will be difficult to achieve. 
Or did I poorly explain what exactly I request

**Comment by afwn90cj93201nixr2e1re** [comment]
trigger_camera

**Comment by LucesEnBabel** [comment]
That can be worked around by increasing the scale of the model.

The bigger the model, the less the impact of looking up/down.

**Comment by CS-PRO1** [comment]
Scaling the model won't give the best results and may 'break' animations in a way.. But really a Cvar for that is needed, and maybe restore HL1's WON movement bobbling and have it as a Cvar as well..

**Comment by Droads2** [comment]
Yes, it can be done with scale on model, but i want disable it for a needed time, not at all. Model with big scale lose all movements in walk, at example. In my case i want to disable movements when player use iron sight
